### PR TITLE
Minor cleanup and sorting imports

### PR DESCRIPTION
**Pull request by jamo**

### CODE REVIEW

File: src/main/java/hy/tmc/core/communication/ExerciseDownloader.java [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package hy.tmc.core.communication;
 
 import com.google.common.base.Optional;
+import static com.google.common.base.Strings.isNullOrEmpty;
```
Comment (nygrenh) [inline src/main/java/hy/tmc/core/communication/ExerciseDownloader.java:3]: Why is this here?

**Comment by jamo** [comment]
Redoing and fixing checkstyle in a separate branch
